### ISSUE TITLE
Rule proposal: `no-unnecessary-ternary`

### ISSUE THREAD

**Issue by MrHen** [sindresorhus/eslint-plugin-unicorn#366]
Ternaries are annoying and due to how truthiness works in JavaScript these two lines of code are functionally equivalent.

## Fail:

```js
const foo = bar ? bar : zaz;
```

## Pass:

```js
const foo = bar || zaz;
```

**Comment by sindresorhus** [comment]
Sounds good.

**Comment by zeorin** [comment]
How is that different from https://eslint.org/docs/rules/no-unneeded-ternary?

**Comment by MrHen** [comment]
Hm, I think @zeorin's right and this is just the same thing as `no-unneeded-ternary`. Early next week, I can double check that that rule would have covered the code I encountered. If it does, I can close down this ticket.

**Comment by fisker** [comment]
Can we close this? @MrHen
